src/libraries/Microsoft.NETCore.Platforms/src/Microsoft.NETCore.Platforms.csproj

```diff
@@ -1,22 +1,40 @@
-<Project Sdk="Microsoft.Build.NoTargets">
+<Project Sdk="Microsoft.NET.SDK">
   <!-- These are wrapper project files for packaging.-->
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.0</TargetFrameworks>
-    <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
+    <TargetFrameworks>$(NetCoreAppToolCurrent);net472</TargetFrameworks>
+    <PackageId>$(MSBuildProjectName)</PackageId>
+    <AvoidRestoreCycleOnSelfReference>true</AvoidRestoreCycleOnSelfReference>
+    <AssemblyName>Microsoft.NETCore.Platforms.BuildTasks</AssemblyName>
```

Comment:
whats the reason behind bringing this infra to runtime ?

Reply:
It’s only used by this single project in runtime.  Splitting it into arcade is unnecessary and complicated changes.  We should delete it from arcade after this. (Note that this helps with our pkgproj removal)